art/command_line.py

```diff
@@ -143,9 +143,6 @@ def install():
 
         # iterate over the zip archive
         for member in archive.infolist():
-            if member.filename.endswith('/'):
-                # skip directories, they will be created as-is
-                continue
```

Comment:
This change could introduce a regression, if someone was relying on `art install` to filter empty directories.

I'm hesitant to add the complexity of a `--keep-empty-dir` option to `art install` for what seems like straight-forward behavior. This only applies when using a "copy all" or "`copy directory" filters.

Reply:
I decided to add the `--keep-empty-dirs` option to preserve the old behavior.

Installing empty directories seems like the correct default, but *needing empty directories* is probably an edge case for actual users of this tool.

*archive*
```
   Date      Time    Attr         Size   Compressed  Name
------------------- ----- ------------ ------------  ------------------------
2023-09-24 10:06:09 D....            0            0  this
2023-09-24 10:06:09 D....            0            0  this/dir
2023-09-24 10:06:09 D....            0            0  this/dir/is
2023-09-24 10:06:09 D....            0            0  this/dir/is/empty
2023-09-24 10:06:09 D....            0            0  this/has
2023-09-24 10:06:09 D....            0            0  this/has/a
2023-09-24 10:06:09 .....            0            5  this/has/a/file
2023-09-24 10:06:09 D....            0            0  this/is
2023-09-24 10:06:09 D....            0            0  this/is/empty
------------------- ----- ------------ ------------  ------------------------
2023-09-24 10:06:09                  0            5  1 files, 8 folders
```

*current/default behavior*
```
$ python3 -m art install
* install: this/has/a/file => artifacts/test/this/has/a/file   -rw-r--r--
```

*--keep-empty-dirs*
```
$ python3 -m art install -k
* install: this/ => artifacts/test/this/   drwxr-xr-x
* install: this/dir/ => artifacts/test/this/dir/   drwxr-xr-x
* install: this/dir/is/ => artifacts/test/this/dir/is/   drwxr-xr-x
* install: this/dir/is/empty/ => artifacts/test/this/dir/is/empty/   drwxr-xr-x
* install: this/has/ => artifacts/test/this/has/   dr-x--x--x
* install: this/has/a/ => artifacts/test/this/has/a/   drwxr-xr-x
* install: this/has/a/file => artifacts/test/this/has/a/file   -rw-r--r--
* install: this/is/ => artifacts/test/this/is/   drwx--x--x
* install: this/is/empty/ => artifacts/test/this/is/empty/   drwxr-xr-x
```